### PR TITLE
RS: PEM encryption & new encryption section/reorg

### PR DESCRIPTION
**Pull request by rrelledge**
DOC-2777

Staged previews:
- [Encrypt private keys/Enable PEM encryption](https://docs.redis.com/staging/DOC-2777/rs/security/encryption/pem-encryption/)
- New [Encryption section index page](https://docs.redis.com/staging/DOC-2777/rs/security/encryption/)

### CODE REVIEW

**Review by yoavredis — APPROVED** [review]

File: content/rs/security/encryption/_index.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+linkTitle: Encryption
+description: Encryption in Redis Enterprise Software.
+weight: 60
+alwaysopen: false
+toc: "true"
+categories: ["RS"]
+aliases: 
+---
+
+Redis Enterprise Software uses encryption to secure communications between clusters, nodes, databases, and clients and protect [data in transit](https://en.wikipedia.org/wiki/Data_in_transit), [at rest](https://en.wikipedia.org/wiki/Data_at_rest), and [in use](https://en.wikipedia.org/wiki/Data_in_use).
```
Comment (kaitlynmichael) [inline content/rs/security/encryption/_index.md:12]:
```suggestion
Redis Enterprise Software uses encryption to secure communications between clusters, nodes, databases, and clients and protects [data in transit](https://en.wikipedia.org/wiki/Data_in_transit), [at rest](https://en.wikipedia.org/wiki/Data_at_rest), and [in use](https://en.wikipedia.org/wiki/Data_in_use).
```

Comment (kaitlynmichael) [reply content/rs/security/encryption/_index.md:12]: You can either change "protect" to "protects" or "to protect"

File: content/rs/security/encryption/_index.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+- Communication from your database to other clusters for replication using [Replica Of]({{<relref "/rs/databases/import-export/replica-of/">}})
+
+- Communication to and from your database to other clusters for [Active-Active]({{<relref "/rs/databases/active-active/_index.md">}}) synchronization
+
+### [Internode encryption]({{<relref "/rs/security/encryption/internode-encryption">}})
+
+Internode encryption uses TLS to encrypt data in transit between cluster nodes.
+
+By default, internode encryption is enabled for the control plane, which manages the cluster and databases. If you also want to encrypt replication and proxy communications between database shards on different nodes, [enable data plane internode encryption]({{<relref "/rs/security/encryption/internode-encryption#enable-data-plane-encryption">}}).
```
Comment (kaitlynmichael) [inline content/rs/security/encryption/_index.md:40]: Great brief explanation of the control plane!

File: content/rs/security/encryption/_index.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+### [Require HTTPS for REST API endpoints]({{<relref "/rs/references/rest-api/encryption">}})
+
+By default, the Redis Enterprise Software API supports communication over HTTP and HTTPS. However, you can [turn off HTTP support]({{<relref "/rs/references/rest-api/encryption">}}) to ensure that API requests are encrypted.
+
+## Encrypt data at rest
+
+### File system encryption
+
+To encrypt data stored on disk, use file system-based encryption capabilities available on Linux operating systems before you install Redis Enterprise Software.
```
Comment (kaitlynmichael) [inline content/rs/security/encryption/_index.md:50]: Bonus points if we add "such as..." with examples of types of encryption.

**Review by kaitlynmichael — APPROVED** [review]
Very well written! Clear and concise.

File: content/rs/security/encryption/_index.md (hunk 1 above)
Comment (rrelledge) [reply content/rs/security/encryption/_index.md]: Fixed

File: content/rs/security/encryption/_index.md (hunk 3 above)
Comment (rrelledge) [reply content/rs/security/encryption/_index.md:50]: Good idea. I think I will open a separate ticket to add this and hopefully some more info/links for client-side encryption.

**Pull request merged**